guides/micronaut-scope-types/kotlin/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
```

Comment:
delete this file. This file will be automatically generated.

Reply:
removed